packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -6172,7 +6172,7 @@ class Listener extends SingleChildRenderObjectWidget {
 ///
 ///  * [Listener], a similar widget that tracks pointer events when the pointer
 ///    has buttons pressed.
-class MouseRegion extends StatefulWidget {
+class MouseRegion extends StatelessWidget {
```

Comment:
I didn't look closely but is it possible that we could remove the _RawMouseRegion entirely? Or rather, remove MouseRegion entirely, and rename _RawMouseRegion to MouseRegion?

It's not clear what purpose there is in having them separate now.

Reply:
ah yes, what was I thinking about. lol. will do that